### ISSUE TITLE
Remove groupchat

### ISSUE THREAD

**Issue by filonzi** [sync0s/i3t#75]

**Comment by skinner904** [comment]
Why? this gives a way for gangs to communicate

**Comment by WalgreensSecurity** [comment]
That's what Radios are for man. The only way people should be communicating is by radio or face to face

**Comment by filonzi** [comment]
Yeah what Wally said. I've noticed people use it to meta-game unfortunately..."Hey lets kill this guy" while they are standing in front of the person or if someone is hiding in a bush and communicating by typing in the group chat so others in the vicinity cant hear them.
